Dockerfile

```diff
@@ -4,6 +4,7 @@ COPY package.json /opt/app/
 RUN apk --no-cache add --virtual native-deps \
   g++ make python && \
   npm install --quiet --production && \
+  npm rebuild node-sass --build-from-source && \
```

Comment:
I don't think this is needed. If there is a build available, we want to use it and not force a rebuild